CMakeLists.txt

```diff
@@ -257,126 +257,95 @@ else()
     if(NOT UMF_LINK_HWLOC_STATICALLY)
         pkg_check_modules(LIBHWLOC hwloc>=2.3.0)
         if(NOT LIBHWLOC_FOUND)
-            find_package(LIBHWLOC 2.3.0 REQUIRED hwloc)
+            find_package(LIBHWLOC 2.3.0 COMPONENTS hwloc)
+            if(LIBHWLOC_LIBRARIES)
+                set(LIBHWLOC_AVAILABLE TRUE)
+            endif()
         endif()
 
-        # add PATH to DLL on Windows
-        set(DLL_PATH_LIST
-            "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_DLL_DIRS}")
-    elseif(WINDOWS)
-        include(FetchContent)
-        set(HWLOC_ENABLE_TESTING OFF)
-        set(HWLOC_SKIP_LSTOPO ON)
-        set(HWLOC_SKIP_TOOLS ON)
+        if(LIBHWLOC_AVAILABLE OR LIBHWLOC_FOUND)
+            # add PATH to DLL on Windows
+            set(DLL_PATH_LIST
+                "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_DLL_DIRS}")
+            message(STATUS "    LIBHWLOC_LIBRARIES = ${LIBHWLOC_LIBRARIES}")
```

Comment:
nit: no need to print this here, it's printed after the `if(UMF_LINK_HWLOC_STATICALLY)` - now it's displayed twice

https://github.com/oneapi-src/unified-memory-framework/actions/runs/13894590051/job/38876218764?pr=1185#step:9:26